src/Grandeljay/Availability/Config.php

```diff
@@ -54,20 +56,122 @@ private function loadConfig(): void
         die('Missing config.json. Please refer to README.md.' . PHP_EOL);
     }
 
+    /**
+     * Processes the passed raw config and returns it in normalized form.
+     *
+     * Note: This function doesn't do any validation.
+     *
+     * @param array $config The raw config to normalize. This is essentially
+     *                      just the decoded json string.
+     *
+     * @return array The normalized config.
+     */
+    private function normalizeConfig(array $rawConfig): array
+    {
+        $normalizedConfig = $rawConfig; // Create a copy.
+
+        $normalizedConfig['directoryAvailabilities'] = $this->normalizeAvailabilitiesDir($rawConfig['directoryAvailabilities']);
+        $normalizedConfig['maxAvailabilitiesPerUser'] = $rawConfig['maxAvailabilitiesPerUser'] ?? 100;
+        $normalizedConfig['defaultDay'] = $rawConfig['defaultDay'] ?? "monday";
+        $normalizedConfig['defaultTime'] = $rawConfig['defaultTime'] ?? "19:00";
+        $normalizedConfig['eventName'] = $rawConfig['eventName'] ?? "Dota 2";
+        $normalizedConfig['logLevel'] = $rawConfig['logLevel'] ?? "Info";
```

Comment:
Can you please align the equal signs to match the code style and use British English (`normalised`) instead?

Reply:
Done